src/main/java/net/snowflake/client/log/SFLogLevel.java

```diff
@@ -0,0 +1,38 @@
+package net.snowflake.client.log;
+
+/** Extended log levels for snowflake package. */
+public enum SFLogLevel {
+  // OFF is highest level, no logs will be shown at this level.
```

Comment:
Please add in the comment, in future we will add level for perf instrumentation and I just realized one for network request/response.